certified-connectors/Tesseron ASM Basic Data/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1085 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Tesseron ASM Basic Data",
+    "description": "This connector allows you to use Tesseron ASM Basic Data API to create, edit and search business partners and contacts. ",
+    "contact": {
+      "name": "Tesseron ASM Support",
+      "email": "development@llnet.de",
+      "url": "https://www.docs.tesseron.de"
+    },
+    "version": "1.0"
+  },
+  "host": "dynamichosturl",
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [],
+  "paths": {
+    "/": {},
+    "/ApiEnterprise/SearchEnterprise": {
+      "post": {
+        "responses": {
+          "default": {
+            "description": "default"
+          },
+          "200": {
+            "description": "success",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "Message": {
+                  "type": "string",
+                  "description": "Message"
+                },
+                "Success": {
+                  "type": "boolean",
+                  "description": "Success"
+                },
+                "EnterpriseId": {
+                  "type": "array",
+                  "items": {
+                    "type": "integer",
+                    "format": "int32"
+                  },
+                  "description": "EnterpriseId"
+                }
+              }
+            }
+          }
+        },
+        "summary": "Search Enterprise ID",
+        "description": "Search the Tesseron ASM enterpriseID with the customer number.",
```

Comment:
@developmentLL strings in CaMel case are considered non-translatable. If you want to leave it as it is, it's fine. If you want the value to be localized later, the casing needs to be fixed.